src/tools/build-manifest/src/main.rs

```diff
@@ -382,14 +379,31 @@ impl Builder {
            .arg(self.input.join(&filename))
            .arg(format!("{}/version", filename.replace(".tar.gz", "")))
            .arg("-O");
-        let version = t!(cmd.output());
-        if !version.status.success() {
+        let output = t!(cmd.output());
+        if !output.status.success() {
             panic!("failed to learn version:\n\n{:?}\n\n{}\n\n{}",
                    cmd,
-                   String::from_utf8_lossy(&version.stdout),
-                   String::from_utf8_lossy(&version.stderr));
+                   String::from_utf8_lossy(&output.stdout),
+                   String::from_utf8_lossy(&output.stderr));
+        }
+        String::from_utf8_lossy(&output.stdout).trim().to_string()
```

Comment:
Maybe I'm not quite following, but why the changes to `version()`? It still should be doing the same thing, right?

Reply:
I changed a variable name before copy/pasting the method, but it’s otherwise unchanged. I can revert that if you prefer.